packages/desktop/views/dashboard/wallet/views/implicit-account-creation-multistep/FundConfirmationView.svelte

```diff
@@ -5,10 +5,16 @@
     import { onMount, onDestroy } from 'svelte'
     import { formatTokenAmountBestMatch, selectedWallet, selectedWalletAssets } from '@core/wallet'
     import { activeProfile } from '@core/profile'
+    import { Balance } from '@iota/sdk/out/types'
 
     // TODO: use this output to calculate mana
     export let outputId: string | undefined
 
+    let walletBalance: Balance | undefined
+
+    $: formattedWalletBalance = walletBalance
+        ? formatTokenAmountBestMatch(Number(walletBalance.baseCoin?.available), baseCoin?.metadata)
+        : '-'
```

Comment:
this makes more sense, right?
```suggestion
    $: formattedWalletBalance = walletBalance?.baseCoin?.available
        ? formatTokenAmountBestMatch(Number(walletBalance.baseCoin.available), baseCoin?.metadata)
        : '-'
```